assets/hole.js

```diff
@@ -38,6 +38,10 @@ let lang;
 let latestSubmissionID = 0;
 
 onload = () => {
+    if (hole != 'quine')
+        for (let info of document.querySelectorAll('.quine'))
+            info.parentNode.removeChild(info);
```

Comment:
I think we can simplify this with [MDN remove](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove):
```javascript
document.querySelectorAll('.quine').forEach(e => e.remove());
```